kitsune/wiki/tests/test_views.py

```diff
@@ -2320,3 +2320,30 @@ def test_custom_locale_mapping_for_header_locale(self):
         trans_content = "This article is translated into fr"
         assert en_content not in doc_content
         assert trans_content in doc_content
+
+
+class PocketArticleTests(TestCaseBase):
+    """Pocket article redirect tests."""
+
+    def setUp(self):
+        ProductFactory()
+
+    def test_pocket_redirect_to_kb_for_document_that_exists(self):
+        """Are we redirecting to our article page?"""
+        doc = DocumentFactory(title="an article title", slug="article-title")
+        RevisionFactory(document=doc, is_approved=True)
+        pocket_style_url = f"/kb/pocket/1111-{doc.slug}"
+        response = self.client.get(pocket_style_url, follow=True)
+        self.assertContains(response, doc.title)
```

Comment:
Can we maybe have an exact match here on the url with `assertEqual`?

Reply:
Sure thing.